drivers/gpio/gpio_pca_series.c

```diff
@@ -1684,11 +1798,6 @@ static int gpio_pca_series_init(const struct device *dev)
 		LOG_ERR("i2c bus device not found");
 		goto out_bus;
 	}
-
-	/** device reset */
-	gpio_pca_series_reset(dev);
-	LOG_DBG("device reset done");
-
```

Comment:
Not entirely sure why this was moved. I can clean it up again on the next revision.

Reply:
Fixed!